src/api/users/controller.js

```diff
@@ -10,7 +10,11 @@ const getUserStats = async (req, res) => {
     try {
         // Get user and their PRs
         const user = await prisma.users.findUnique({
```

Comment:
Did you test it? 
it should be findFirst, findUnique won't work in this case.

Reply:
Uhh my bad!

Reply:
Somehow forgot to update it...

Reply:
Yup, nw. Fixed it.